### PR TITLE
Phish.AI: Bugfix with api key usage

### PR DESCRIPTION
**Pull request by yevgenypats**

### CODE REVIEW

File: Integrations/integration-PhishAI.yml [hunk 1]
```diff
@@ -7,15 +7,21 @@ category: Data Enrichment & Threat Intelligence
...
+  name: apiKey
   defaultvalue: ""
   type: 0
   required: false
+- display: Use system proxy settings
+  name: proxy
+  defaultvalue: "false"
+  type: 8
+  required: false
 script:
```
Comment (hirshbergshachar) [inline Integrations/integration-PhishAI.yml]: Need to add an "insecure" param

File: Integrations/integration-PhishAI.yml [hunk 2]
```diff
@@ -7,15 +7,21 @@ category: Data Enrichment & Threat Intelligence
...
   script: |2
 
     var TIMEOUT = 20 * 1000;
+    var DONT_CHECK_CERTIFICATE = false;
```
Comment (hirshbergshachar) [inline Integrations/integration-PhishAI.yml]: No need if we have an "insecure" param

File: Integrations/integration-PhishAI.yml [hunk 3]
```diff
@@ -41,7 +47,8 @@ script:
                 },
                 Body: JSON.stringify({'url': url})
             },
-            "false"
+            DONT_CHECK_CERTIFICATE,
```
Comment (hirshbergshachar) [inline Integrations/integration-PhishAI.yml]: Change to params.insecure

File: Integrations/integration-PhishAI.yml [hunk 4]
```diff
@@ -54,20 +61,22 @@ script:
...
                     Method: 'GET',
                     Headers: {
-                        'Accept': ['application/json']
+                        'Accept': ['application/json'],
+                        'Authorization': [apiKey]
                     }
                 },
-                "false"
+                DONT_CHECK_CERTIFICATE,
+                params.proxy
```
Comment (hirshbergshachar) [inline Integrations/integration-PhishAI.yml]: Change to params.insecure

**Comment by anara123** [comment]
@hirshbergshachar I think in this case when the Phish.AI service is SAAS, we can always assume that certificate could and should be validated. I think this can be ok to leave it constant

**Pull request merged**